docs/cryptodoc/src/05_06_hss_lms.rst

```diff
@@ -0,0 +1,416 @@
+   +=====================+==============+=====+=====+=====+=====+========+
+   | LMOTS_SHA256_N32_W1 | SHA-256      | 32  | 1   | 265 | 7   | 0x0001 |
+   +---------------------+--------------+-----+-----+-----+-----+--------+
+   | LMOTS_SHA256_N32_W2 | SHA-256      | 32  | 2   | 133 | 6   | 0x0002 |
+   +---------------------+--------------+-----+-----+-----+-----+--------+
+   | LMOTS_SHA256_N32_W4 | SHA-256      | 32  | 4   | 67  | 4   | 0x0003 |
+   +---------------------+--------------+-----+-----+-----+-----+--------+
+   | LMOTS_SHA256_N32_W8 | SHA-256      | 32  | 8   | 34  | 0   | 0x0004 |
+   +---------------------+--------------+-----+-----+-----+-----+--------+
+   | LMOTS_SHA256_N24_W1 | SHA-256/192  | 24  | 1   | 200 | 8   | 0x0005 |
+   +---------------------+--------------+-----+-----+-----+-----+--------+
+   | LMOTS_SHA256_N24_W2 | SHA-256/192  | 24  | 2   | 101 | 6   | 0x0006 |
+   +---------------------+--------------+-----+-----+-----+-----+--------+
+   | LMOTS_SHA256_N24_W4 | SHA-256/192  | 24  | 4   | 51  | 4   | 0x0007 |
+   +---------------------+--------------+-----+-----+-----+-----+--------+
+   | LMOTS_SHA256_N24_W8 | SHA-256/192  | 24  | 8   | 26  | 0   | 0x0008 |
+   +---------------------+--------------+-----+-----+-----+-----+--------+
+   | LMOTS_SHAKE_N32_W1  | SHAKE256/256 | 32  | 1   | 265 | 7   | 0x0009 |
+   +---------------------+--------------+-----+-----+-----+-----+--------+
+   | LMOTS_SHAKE_N32_W2  | SHAKE256/256 | 32  | 2   | 133 | 6   | 0x000a |
+   +---------------------+--------------+-----+-----+-----+-----+--------+
+   | LMOTS_SHAKE_N32_W4  | SHAKE256/256 | 32  | 4   | 67  | 4   | 0x000b |
+   +---------------------+--------------+-----+-----+-----+-----+--------+
+   | LMOTS_SHAKE_N32_W8  | SHAKE256/256 | 32  | 8   | 34  | 0   | 0x000c |
+   +---------------------+--------------+-----+-----+-----+-----+--------+
+   | LMOTS_SHAKE_N24_W1  | SHAKE256/192 | 24  | 1   | 200 | 8   | 0x000d |
+   +---------------------+--------------+-----+-----+-----+-----+--------+
+   | LMOTS_SHAKE_N24_W2  | SHAKE256/192 | 24  | 2   | 101 | 6   | 0x000e |
+   +---------------------+--------------+-----+-----+-----+-----+--------+
+   | LMOTS_SHAKE_N24_W4  | SHAKE256/192 | 24  | 4   | 51  | 4   | 0x000f |
+   +---------------------+--------------+-----+-----+-----+-----+--------+
+   | LMOTS_SHAKE_N24_W8  | SHAKE256/192 | 24  | 8   | 26  | 0   | 0x0010 |
+   +---------------------+--------------+-----+-----+-----+-----+--------+
+
+In addition to these parameters, an LM-OTS
+instance is defined by the identifier ``I`` of the LMS tree
+and the index of its leaf ``q``, where the LM-OTS instance is located; this is
+represented by the class ``OTS_Instance``.
+For each LM-OTS instance, we can create a keypair with a secret key (class
+``LMOTS_Private_Key``) and a public key (class ``LMOTS_Public_Key``). The
+pseudorandom key generation method recommended in [RFC8554]_ Appendix A derives the
```

Comment:
Ok, I'll add a reference to SP800-208. However, since this document is called _Recommendation for Stateful Hash-Based Signature Schemes_, the term "recommended" should be fine.

Reply:
I looked it up again to be sure, and only found a *shall* in *6.1 LMS and HSS Random Number Generation Requirements* of the NIST document for this.

So I was wrong, Probably still good to keep the reference to SP800-208, but make it recommended only.